File: Python/Casting/Casting_main.py
import Python.Casting.Casting_doubles as ds
import Python.Casting.Casting_movie as mv

def main():
    ds = ds.Casting_doubles()
    mv = mv.Casting_movie()

    option = 0

    while option != 3:
        print("Welcome to Casting")
        print("What would you like to see")
        print("1: Print Doubles")
        print("2: Print Rounded Doubles")
        print("3: Enter Movie Rating")

        option = input('You choice: ')

        switcher =
        {
            1: ds.pNormalVals(),
            2: pRoundVals(),
            3: mv.rating(),
            4: exit()

        }

if __name__ == "__main__":
    main()

File: Java/Casting/Casting_main.java
package Java.Casting;

import java.util.Scanner;

public class Casting_main 
{
    public static void main(String[] args) 
    {
        Casting_doubles ds = new Casting_doubles();
        Casting_movie mv = new Casting_movie();

        Scanner scan_numbers = new Scanner(System.in);

        int option=0;
        while (option!=3)
        {
            System.out.println("Welcome to Casting");
            System.out.println("What would you like to see");
            System.out.println("1: Print Doubles");
            System.out.println("2: Print Rounded Doubles");
            System.out.println("3: Enter Movie Rating");

            option = scan_numbers.nextInt();

            switch (option) 
            {
                case 1:
                    ds.pNormVals();
                    break;
                case 2:
                    ds.pRoundVals();
                    break;
                case 3:
                    mv.rating();
                    break;
                case 4:
                    System.exit(0);
                    break;
                default:
                    System.out.println("1: Print Doubles");
                    System.out.println("2: Print Rounded Doubles");
                    System.out.println("3: Enter Movie Rating");
                    System.out.println("4: Done");
                    break;
            }
        }
			
    }
    
}